### PR TITLE
Style/clippy 20210914

### PR DESCRIPTION
**Pull request by escapedcat**
Remove more 📎  warnings   
before: `172 warnings`
after: `104 warnings`

### CODE REVIEW

File: src/bin/spammer.rs [hunk 1]
```diff
@@ -63,10 +63,7 @@ pub async fn main() -> saito_rust::Result<()> {
         )
         .get_matches();
 
-    let config_name = match matches.value_of("config") {
-        Some(name) => name,
-        None => "config",
-    };
+    let config_name = matches.value_of("config").unwrap_or("config");
```
Comment (clayrab) [inline src/bin/spammer.rs:66]: Oh, clippy is very nice...

File: src/networking/handlers.rs [hunk 2]
```diff
@@ -127,7 +127,7 @@ pub async fn get_block_handler(str_block_hash: String) -> Result<impl Reply> {
 //         Ok(json_data) => Ok(warp::reply::json(&json_data)),
 //         Err(_err) => {
 //             eprintln!("{:?}", _err);
-//             return Err(warp::reject());
+//             Err(warp::reject())
```
Comment (clayrab) [inline src/networking/handlers.rs:130]: wait, what?

File: src/staking.rs [hunk 3]
```diff
@@ -158,14 +158,14 @@ impl Staking {
             let (z, f) = nominator.overflowing_div(denominator);
 
             let mut staking_profit: u64 = 0;
-            if f != true {
+            if !f {
```
Comment (clayrab) [inline src/staking.rs:161]: what the f?

**Review by clayrab — APPROVED** [review]
lgtm.

<3 Clippy 📎

File: src/bin/spammer.rs (hunk 1 above)
Comment (escapedcat) [reply src/bin/spammer.rs:66]: show sum luvvv for ze 📎  !

File: src/networking/handlers.rs (hunk 2 above)
Comment (escapedcat) [reply src/networking/handlers.rs:130]: let's fix commented out code!

File: src/staking.rs (hunk 3 above)
Comment (escapedcat) [reply src/staking.rs:161]: exactly!

**Pull request merged**